interfaces/Glitter/templates/include_history.tmpl

```diff
@@ -184,7 +184,9 @@
                         </div>
                         <!-- /ko -->
                     </div>
-                    <a href="#" class="hover-button" data-bind="click: deleteSlot, css: { disabled : processingDownload() || processingWaiting() }"><span class="glyphicon glyphicon-trash"></span></a>
+                    <a href="#" data-bind="click: deleteSlot">
```

Comment:
Is it possible to terminate the running job if you abort (shutdown or ^C) SABnzbd?